src/coreclr/inc/crsttypes.h

```diff
@@ -84,57 +84,59 @@ enum CrstType
     CrstModule = 66,
     CrstModuleFixup = 67,
     CrstModuleLookupTable = 68,
-    CrstMulticoreJitHash = 69,
-    CrstMulticoreJitManager = 70,
-    CrstNativeImageEagerFixups = 71,
-    CrstNativeImageLoad = 72,
-    CrstNls = 73,
-    CrstNotifyGdb = 74,
-    CrstObjectList = 75,
-    CrstPEImage = 76,
-    CrstPendingTypeLoadEntry = 77,
-    CrstPgoData = 78,
-    CrstPinnedByrefValidation = 79,
-    CrstProfilerGCRefDataFreeList = 80,
-    CrstProfilingAPIStatus = 81,
-    CrstRCWCache = 82,
-    CrstRCWCleanupList = 83,
-    CrstReadyToRunEntryPointToMethodDescMap = 84,
-    CrstReflection = 85,
-    CrstReJITGlobalRequest = 86,
-    CrstRetThunkCache = 87,
-    CrstSavedExceptionInfo = 88,
-    CrstSaveModuleProfileData = 89,
-    CrstSecurityStackwalkCache = 90,
-    CrstSigConvert = 91,
-    CrstSingleUseLock = 92,
-    CrstSpecialStatics = 93,
-    CrstStackSampler = 94,
-    CrstStressLog = 95,
-    CrstStubCache = 96,
-    CrstStubDispatchCache = 97,
-    CrstStubUnwindInfoHeapSegments = 98,
-    CrstSyncBlockCache = 99,
-    CrstSyncHashLock = 100,
-    CrstSystemBaseDomain = 101,
-    CrstSystemDomain = 102,
-    CrstSystemDomainDelayedUnloadList = 103,
-    CrstThreadIdDispenser = 104,
-    CrstThreadpoolTimerQueue = 105,
-    CrstThreadpoolWaitThreads = 106,
-    CrstThreadpoolWorker = 107,
-    CrstThreadStore = 108,
-    CrstTieredCompilation = 109,
-    CrstTypeEquivalenceMap = 110,
-    CrstTypeIDMap = 111,
-    CrstUMEntryThunkCache = 112,
-    CrstUMEntryThunkFreeListLock = 113,
-    CrstUniqueStack = 114,
-    CrstUnresolvedClassLock = 115,
-    CrstUnwindInfoTableLock = 116,
-    CrstVSDIndirectionCellLock = 117,
-    CrstWrapperTemplate = 118,
-    kNumberOfCrstTypes = 119
+    CrstMonoHandles = 69,
+    CrstMonoICalls = 70,
+    CrstMulticoreJitHash = 71,
```

Comment:
Will changing the value of `CrstMulticoreJitHash` (and maybe others) cause adverse affects? Would they accept this upstream?

Reply:
It is fine to add new entries to CrstTypes.def and regenerate CrstTypes.h.

Reply:
Great thanks!